config/initializers/secure_headers.rb

```diff
@@ -14,11 +12,17 @@
   config.x_content_type_options = "nosniff"
 
   # http://msdn.microsoft.com/en-us/library/dd565647%28v=vs.85%29.aspx
-  config.x_xss_protection = { :value => 1 }
+  config.x_xss_protection = "1; mode=block"
 
   # https://w3c.github.io/webappsec/specs/content-security-policy/
-  config.csp = false
+  config.csp = {
```

Comment:
Should we be using `SecureHeaders::OPT_OUT` here to match the previous behaviour?